app/components/ship_order_component.html.haml

```diff
@@ -1,6 +1,6 @@
 = render ConfirmModalComponent.new(id: "ship_order", confirm_reflexes: "click->Admin::OrdersReflex#ship", controller: "orders", reflex: "Admin::Orders#ship") do
   %div{class: "margin-bottom-30"}
-    %p= t('spree.admin.orders.shipment.mark_as_shipped_message')
+    %p= t('spree.admin.orders.shipment.mark_as_shipped_message').html_safe
```

Comment:
I would split the two lines into two different translations so that you can keep the HTML in the view.

Otherwise, you could also suffix the translation key with `_html` and then you don't need to mark it as html_safe.

Reply:
ah I did not know adding a _html suffix marks the translation as html_safe. thanks!